src/utils/BrambleExtensionLoader.js

```diff
@@ -60,7 +61,8 @@ define(function (require, exports, module) {
     var extraExtensions = [
         "brackets-cdn-suggestions",    // https://github.com/szdc/brackets-cdn-suggestions
         "ImageUrlCodeHints",
-        "HTMLHinter"
+        "HTMLHinter",
+        "Autosave"
```

Comment:
I think the bramble extension is getting too big, and I wanted an easy way to turn this feature on/off for user testing.